com.unity.render-pipelines.universal/Runtime/2D/Light2DShape.cs

```diff
@@ -34,26 +29,13 @@ sealed public partial class Light2D : MonoBehaviour
         //                              Functions
         //==========================================================================================
 
-        internal bool IsShapeLight()
-        {
-            return m_LightType != LightType.Point;
-        }
+        internal bool isShapeLight => m_LightType != LightType.Point;
 
-        BoundingSphere GetShapeLightBoundingSphere()
+        float GetShapeLightBoundingSphere(Bounds bounds)
```

Comment:
rename this to `CalculateBoundingSphereRadius`

Reply:
Fixed.